docs/topics_subscribers.md

```diff
@@ -0,0 +1,332 @@
+---
+sidebar_position: 8
+tags: ['distributed system', 'fault-tolerant', 'event-driven', 'microservice', 'pub/sub', 'topic', 'event', 'queue', 'async', 'publisher', 'subscriber', 'processor']
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Topics / Subscribers (Pub / Sub)
+
+Less makes it easy to set up topics and subscribers (pub/sub) giving you powerful features by default:
+- Publish messages with at-least-once delivery guarantees.
+- Failed messages are retained by Less in order to be processed again later.
+- Failed messages to not block the processing of subsequent messages.
+- Easily implement the Fanout arquitectural pattern.
+- Publish and subscribe to topics from different Less applications securely.
+- Easily build event-driven microservices.
+- Generated Topics REST API allowing you to publish messages from anywhere.
+- Easily integrate with your existing systems.
+- Easily scale and distribute your workloads.
+
+## Create Topics
+
+In order to create topics (publishers) and subscribers you need to create a `topics` folder in `less`. Each folder inside of `topics` is your topic.
+
+```bash
+mkdir -p less/topics
+```
+
+Let's create a `user_created` topic:
+```bash
+mkdir less/topics/user_created
+```
+
+### Create Topic Subscribers
+A topic can have several subscribers. Just add them to your topic's folder.
+
+Let's create some example subscribers to the `user_created` event:
+```bash
+mkdir less/topics/user_created/send_welcome_email
+mkdir less/topics/user_created/send_to_analytics
+```
+
+<Tabs groupId="programming-language" queryString="programming-language">
+  
+  <TabItem value="nodejs" label="Node.js">
+    ```bash
+    touch less/topics/user_created/send_welcome_email/index.js
+    ```
+    
+    ```js title="less/topics/user_created/send_welcome_email/index.js" showLineNumbers
+    exports.process = async (user) => {
+      console.log(`Sending a welcome email to ${user.email}`);
+    };
+    ```
+
+    ```js title="less/topics/user_created/send_to_analytics/index.js" showLineNumbers
+    exports.process = async (user) => {
+      console.log(`Sending user "${user.name}" to analytics`);
+    };
+    ```
+  </TabItem>
+  
+</Tabs>
+
+## Send Messages to Topics
+Less offers 2 ways to send messages to topics:
+1. Using the `topics` module in the Less SDK.
+2. Using the Less Topics REST API.
+
+### Send Messages Using the SDK
+Use the Less `topics` module to send messages to all of a topic's subscribers.
+
+<Tabs groupId="programming-language" queryString="programming-language">
+  <TabItem value="nodejs" label="Node.js">
+    ```js showLineNumbers
+    const { topics } = require('@chuva.io/less');
+
```

Comment:
I think you can create user object here like ex: `const user = {name: "John", email: "John@exampe.com"};` use it on the publish method bellow to be more clear an consistent with what the processor created above are expecting

Reply:
I don't want to add it here because I just want to show the function and import. There is example code below.